core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -82,7 +82,7 @@ object ReassignPartitionsCommand extends Logging {
     }
   }
 
-  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions) {
+  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions): Map[TopicAndPartition, Seq[Int]] = {
```

Comment:
It is more of a style issue. I noticed that most of the command class' `main` does not have println.

The above PR refactored the `generatedAssignement` function into two and there is no println in main. I am fine with this approach.